lib/cm/cm.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Created by kugajjar on 6/3/15.
+ */
+
+'use strict';
+
+var affiliate_offer         = require('../../providers/affiliate_offer.json').affiliate_offer,
+    Q                       = require('q'),
+    utils                   = require('../utils/utils'),
+    client                  = require('./client');
+
+require('date-utils');
+
+module.exports = function cm(nemo) {
+
+    var cm_config = nemo._config.get('cm');
+
+    var setup_affiliate_offer = function () {
```

Comment:
I think this is what we should be doing instead of our giant JSON files for our mock adapters.